guides/source/active_record_validations.md

```diff
@@ -188,13 +193,13 @@ false` as an argument. This technique should be used with caution.
 [`upsert`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert
 [`upsert_all`]: https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-upsert_all
 
-### `valid?` and `invalid?`
+### Checking Validity
```

Comment:
I think this is a good change, describing the use-case instead of the methods.
Maybe we should add this to the guidelines as well?

Reply:
Done ✔️